.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push]
```

Comment:
@doodlesbykumbi this won't give us daily runs - can you please add a cronjob for this like we have here: 
https://github.com/cyberark/conjur-quickstart/blob/76dbdbf463c3f4405c4c87dc2607eb89b458c25d/.github/workflows/quick_start_flow_test.yml#L5